### DISCUSSION TITLE
rlink 2 antitheft fix

### DISCUSSION THREAD

**Discussion by Vuk031** [cedricp/ddt4all#1297] [Q&A]
I have Renault Scenic 4
On device r link 2 on screen i have antitheft.
I virgined device with ddt, after that r link changed the vin number ( same as car vin) but i have same problem with antitheft on screen.
can anyone fix it?

**Comment by KarelSvo** [comment]
How is someone supposed to help without seeing your ddt4all log files?

**Comment by Timmytubby** [comment]
> I have Renault Scenic 4
> 
> On device r link 2 on screen i have antitheft.
> 
> I virgined device with ddt, after that r link changed the vin number ( same as car vin) but i have same problem with antitheft on screen.
> 
> can anyone fix it?

How do you Virgin it? Can figure out the procedure.

**Comment by AlMeu** [comment]
@Vuk031 
I can remove the antitheft code but I need the unit on my desk.

**Comment by BBFul** [comment]
AlMeu, how do you do that?

**Comment by KarelSvo** [comment]
save dump, ecu virgin , in the dump switch off anti theft and change vin , load dump  
or not ?

**Comment by BBFul** [comment]
Hi @KarelSvo! can you send me your email to send you the .bin file? Thanks!

**Comment by Pekora78** [comment]
Buenas tardes. tengo 2  r-link 2  con el antirrobo activado ¿alguien tiene alguna solución?

**Comment by AlMeu** [comment]
@Pekora78 
An email and maybe I can help you.

**Comment by Pekora78** [comment]
daviciyo_78@hotmail.com este es mi correo y estaría muy agradecido por tu ayuda.

**Comment by Vuk031** [comment]
> @Pekora78 An email and maybe I can help you.

Can  you send me email, maybe you can help me with antitheft on Rlink2
thanks

**Comment by florin2025** [comment]
Hello.
can you send an email, maybe you can help me with an antithelf code.
thank you

**Comment by AlMeu** [comment]
> @Vuk031 I can remove the antitheft code but I need the unit on my desk.

@florin2025 leave your email.

**Comment by florin2025** [comment]
florin_2013@yahoo.com thanks

**Comment by Arnaudkadjar** [comment]
Can someone help me with anti theth locked in R link Renault Kadjar , the rlink was bad ,so I had to install a second hand. I ve tried to change VIN with renolink but no way, it doesn t change. Thank a lot

**Comment by AlMeu** [comment]
@Arnaudkadjar leave an email and perhaps I can help you.

**Comment by Arnaudkadjar** [comment]
HI .

This is my email
***@***.***
Thank a lot
Regards

**Comment by AlMeu** [comment]
@Arnaudkadjar you have to login on site for leave an email.

**Comment by Arnaudkadjar** [comment]
My email adress 
Arnaud.gugnot@gmail.com

**Comment by AlMeu** [comment]
@Zener-hub leave an email and we can talk.

**Comment by lklik** [comment]
Hi AIMeu i have the same problem this is my email ; azizouaankour@yahoo.fr

**Comment by gmzr0** [comment]
> @Zener-hub leave an email and we can talk.

is there any chance u could help me?

**Comment by Goran-Barznje** [comment]
Hi, someone help for same issues after replaced the nav radio for Rlink2.
goran_barznje@yahoo.com

**Comment by JohnnyX99** [comment]
Hello someone figured out how to virgin and deactivate theft mode ?

**Comment by saadousfar** [comment]
i have the solution no need to dissamble the unit , unlock done by can 
![rlink2](https://github.com/user-attachments/assets/94af330b-c03c-495c-8358-6851a3b79094)

**Reply by saadousfar** [reply]
contact : saadousfar59@gmail.com

**Comment by saadousfar** [comment]
> I have Renault Scenic 4 On device r link 2 on screen i have antitheft. I virgined device with ddt, after that r link changed the vin number ( same as car vin) but i have same problem with antitheft on screen. can anyone fix it?

i have the solution no need to read the eeprom you can unlock any Rlink2 unit by obd via my script for more information contact me on saadousfar59@gmail.com
